src/parser/druid/restoration/modules/Abilities.js

```diff
@@ -182,6 +182,16 @@ class Abilities extends CoreAbilities {
           SPELLS.LIFEBLOOM_BLOOM_HEAL.id,
         ],
       },
+      {
+        spell: SPELLS.LIFEBLOOM_DTL_HOT_HEAL,
```

Comment:
Just thought about it but you need to check if they have DTL on here and on normal life bloom or you get this 
![image](https://user-images.githubusercontent.com/25177817/103150225-00b0d500-4740-11eb-8286-40cd49652b18.png)